apps/coa/sections/changeDetails-data-sections.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  'Home address details': {
```

Comment:
naming-convention for object keys would be more convenient to use later for multi-language translations, for example.

Reply:
changes done